File-Formats/XlsIO/Working-with-Template-Markers.md

```diff
@@ -257,7 +257,7 @@ using (ExcelEngine excelEngine = new ExcelEngine())
 
 The following screenshot represents generated Excel file in which the array of data is bounded.
 
-<img src="Working-with-Template-Markers_images/Bind_data_from_Array_to_Excel_Spreadsheet_Template.jpeg" alt="Bind from Array Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/bind-data-from-array-to-excel-spreadsheet-template.jpeg" alt="Bind data from array to spreadsheet using XLSIO template marker" width="100%" Height="Auto"/>
```

Comment:
spreadsheet and xlsio both are same.
top one you are using xlsio, here use spreadsheet aone. Don't use both keywords in same place.
I mean use spreadsheet template marker

Reply:
Corrected